### PR TITLE
Move Slot and Epoch functions from helpers to core pkg

### PR DESCRIPTION
**Pull request by prestonvanloon**
**What type of PR is this?**

Other

**What does this PR do? Why is it needed?**

The "helpers" package has become a catch-all for lost functions. This PR moves pure slot and epoch functions that do not require any cache or other dependencies.

**Which issues(s) does this PR fix?**

**Other notes for review**

### CODE REVIEW

**Comment by prestonvanloon** [comment]
Fixing the "core2" imports...

**Review by rauljordan — DISMISSED** [review]

**Review by rauljordan — APPROVED** [review]

File: beacon-chain/blockchain/head_sync_committee_info.go [hunk 1]
```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/pkg/errors"
 	types "github.com/prysmaticlabs/eth2-types"
 	"github.com/prysmaticlabs/prysm/beacon-chain/cache"
+	core2 "github.com/prysmaticlabs/prysm/beacon-chain/core"
```
Comment (terencechain) [inline beacon-chain/blockchain/head_sync_committee_info.go:9]: core2? Goland auto import failed :(

File: beacon-chain/blockchain/process_attestation_helpers.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/pkg/errors"
 	types "github.com/prysmaticlabs/eth2-types"
+	core2 "github.com/prysmaticlabs/prysm/beacon-chain/core"
```
Comment (terencechain) [inline beacon-chain/blockchain/process_attestation_helpers.go:10]: Same here

**Review by terencechain — CHANGES_REQUESTED** [review]
please fix all `core2` instances

**Pull request merged**